code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -475,29 +475,31 @@
 
 // Handles mob movement inside a pipenet
 /obj/machinery/atmospherics/relaymove(mob/living/user, direction)
-	direction &= initialize_directions
```

Comment:
why was this removed?

Reply:
no idea why was added in the first place, this should check if the direction we are moving to has a connected machine. I did all the tests and this line is totally not needed

Reply:
Doesn't this line prevent moving into something that you don't share a corner with? Does it ever cause a return if you keep it there?

Reply:
i just tested it and readding the line breaks the code and causes multiple not needed calls (since the smart pipes are ALL_CARDINAL you check all sides when moving) and also you can't move up and down when it's present